SQL/Archive/2016-10-02-FixAccessProfilePermissions.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM LorisMenuPermissions WHERE MenuID=(SELECT ID FROM LorisMenu WHERE Label='Access Profile' AND Parent=1) AND PermID=(SELECT PermID FROM permissions WHERE code='data_entry');
```

Comment:
This remove the menu item for user that don't have access_all_profile. (Those that have data_entry only)